packages/labs/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "lit-labs",
+  "version": "1.0.0-pre.1",
+  "description": "An experimental playground for Lit helpers and tools. Your mileage may vary.",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit-element.polymer-project.org/",
+  "repository": "lit-html",
+  "type": "module",
+  "main": "index.js",
+  "module": "index.js",
+  "typings": "index.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "exports": {
+    ".": {
+      "development": "./development/index.js",
+      "default": "./index.js"
+    }
+  },
+  "files": [
+    "/src/",
+    "!/src/test/",
+    "/index.d.ts",
+    "/index.d.ts.map",
+    "/index.js",
+    "/index.js.map"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsc && rollup -c",
+    "clean": "rm -rf index.{js,js.map,d.ts} development/",
+    "dev": "scripts/dev.sh",
+    "build:ts": "tsc",
+    "build:ts:watch": "tsc --watch",
+    "test": "npm run test:dev && npm run test:prod",
+    "test:dev": "cd ../tests && npx wtr '../labs/development/**/*_test.js'",
+    "test:prod": "TEST_PROD_BUILD=true npm run test:dev",
+    "test:watch": "npm run test:dev -- --watch",
+    "checksize": "rollup -c --environment=CHECKSIZE",
+    "format": "prettier src/* --write",
+    "lint": "tslint --project ./",
```

Comment:
```suggestion
```

Now the format and lint scripts are on the root `package.json`.

Reply:
Done.